cmd/generate/testdata/success_manifest.golden

```diff
@@ -2,27 +2,27 @@
     "flags": {
       "enableFeatureA": {
         "flagType": "boolean",
-        "defaultValue": false,
+        "codeDefault": false,
```

Comment:
I think we should revert this.

I understand why you'd change it, `defaultValue` seems a bit confusing right next to `defaultVariant` (in flagd, for example) but I think that's just because there's a bit of a conceptual hurdle around "code default" vs "no targeting matched fallthrough" anyway, and that hurdle is somewhat inevitable.

"defaultValue" is what this parameter is called in code, and I think for consistency we should use that here, or we risk even more confusion.

Reply:
Okay, good point. I've reverted this change. Please re-review when you have a moment.